README.md

```diff
@@ -214,6 +216,33 @@ customMetric(String name, long value)
 Calling either of these will add a custom metric with the specified name and
 the given value. Custom metric names are limited to 128 characters.
 
+## Event Info
+
+This plugin records input event types and includes in the report the origin
+of certain events.
+
+It operates on the given input classes:
+
+ * `com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent`
+ * `com.amazonaws.services.lambda.runtime.events.CloudFrontEvent`
+ * `com.amazonaws.services.lambda.runtime.events.KinesisEvent`
+ * `com.amazonaws.services.lambda.runtime.events.KinesisFirehoseEvent`
+ * `com.amazonaws.services.lambda.runtime.events.S3Event`
+ * `com.amazonaws.services.lambda.runtime.events.ScheduledEvent`
+ * `com.amazonaws.services.lambda.runtime.events.SNSEvent`
+ * `com.amazonaws.services.s3.event.S3EventNotification`
+
+By default this plugin is enabled and requires no changes to your code unless
+you are using IOpipe via the manual method. To disable the plugin you may set
+the environment variable `IOPIPE_EVENT_INFO_ENABLE` to `false`.
```

Comment:
I think this should be `ENABLED` instead of `ENABLE` to be consistent with `IOPIPE_ENABLED`.

Reply:
I'm good with this update

Reply:
This change would affect all plugins and the documentation, not just event info. Would you want all those changes in the event-info PR or would you want a new PR? I ask this so that way those changes are not out of scope and that confusion is not created from it.

Reply:
@kolanos another PR sound ok with you? I appreciate keeping the PRs smaller in scope.